spec.go

```diff
@@ -83,6 +83,12 @@ var specCommand = cli.Command{
 						Destination: "/sys",
 						Options:     "nosuid,noexec,nodev",
 					},
+					{
+						Type:        "cgroup",
+						Source:      "cgroup",
+						Destination: "/sys/fs/cgroup",
+						Options:     "nosuid,noexec,nodev,relatime,ro",
```

Comment:
I have question, shouldn't we mount `/sys/fs/cgroup` as `ro` too? because now we mount only subsystems as `ro`.